app/presenters/education_navigation_ab_testable.rb

```diff
@@ -0,0 +1,17 @@
+module EducationNavigationABTestable
```

Comment:
Yes I've thought about this approach. I decided against it mainly because of verbosity. Any controllers using this class would have to:
* Instantiate the class
* Attach `helper_method` to make methods available on view
* Attach `after_filter` to mutate the `response` object

I think my ideal API (as discussed with @tijmenb) would be to have a method similar to things like `helper_method` etc., and declare at the top of the controller: `ab_test :educationnavigation`.

However, this is all pretty temporary code, so I don't think we should worry about it too much. Let me know if you think differently.